src/quic/node_quic_buffer.h

```diff
@@ -116,7 +116,7 @@ class QuicBuffer : public MemoryRetainer {
 
   ~QuicBuffer() override {
     Cancel();  // Cancel the remaining data
-    CHECK_EQ(length_, 0);
+    DCHECK_EQ(length_, 0);
```

Comment:
I’d prefer to keep this as is, at least as long as the QUIC code is under active development